### ISSUE TITLE
Custom ignore pattern

### ISSUE THREAD

**Issue by ericcornelissen** [ericcornelissen/goparamcount#2]
# Feature request

- version: `v0.1.0`

## Description

Currently, the program will [ignore some files and folders](https://github.com/ericcornelissen/goparamcount/blob/61e89db71417c8c8aa0854120ab9411752c8553d/main.go#L75-L92) by default. It would be nice if this can be configured manually.

## Proposal

Add a new CLI flag called `-exclude` of type **string** that can be used to specify a regular expression of files that should be ignored. For example

```shell
$ goparamcount -exclude *_fuzz.go ./...
```

Should run *goparamcount*  on the current folder and it's subfolder, ignoring any files that have the suffix `_fuzz.go`.

In addition, it should be possible to provide multiple patterns using a comma separated list. For example

```shell
$ goparamcount -exclude foo*.go,*bar.go ./...
```

Should run *goparamcount*  on the current folder and it's subfolder, ignoring any files that have the suffix `bar.go` or match `foo*.go`.

**Comment by ericcornelissen** [comment]
Following the [go Magic Number Detector](https://github.com/tommy-muehle/go-mnd) I want to propose we name this flag `-exclude` instead of `-ignore`.

**Comment by ericcornelissen** [comment]
Closed with 5f7c241b859476f6d3bd1c3f4ad16ff23798692d
